src/app/channels/channel/channel.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit, OnDestroy, OnChanges } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { ChannelService } from '../channel.service';
+import { Channel } from '../channel.model';
+import { PatternGeneratorService } from '../../shared/pattern-generator.service';
+import { UserService } from '../../shared/user.service';
+
+@Component({
+  selector: 'bs-channel',
+  templateUrl: './channel.component.html',
+  styleUrls: ['./channel.component.scss']
+})
+export class ChannelComponent implements OnInit, OnChanges, OnDestroy {
+  routeIdSubscription: any;
+  channel: Partial<Channel>;
+  headerImage: any;
+  isUserTheOwner: boolean = false;
```

Comment:
Since when do we add `The` to variable names? :laughing: 
Change to `isUserOwner`

Reply:
Without the "The" the var could be meaning that the user is an owner of a channel, but not this specific channel.

Reply:
Done.